### PR TITLE
Add a Constants value and util method to getIDByName

### PR DESCRIPTION
**Pull request by Jeff-Lv**
## What is the purpose of the change

Support the feature of alibaba inside's HSF which remoting protocol will be opensources to Dubbo.

## Brief changelog

add a constants value 
a new method for CodeSupporet

### CODE REVIEW

File: dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java [hunk 1]
```diff
@@ -363,6 +363,8 @@
 
     public static final String DUBBO_VERSION_KEY = "dubbo";
 
+    public static final String DEFAULT_VERSION = "0.0.0";
```
Comment (beiwei30) [inline dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: why should we introduce this?

File: dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/CodecSupport.java [hunk 2]
```diff
@@ -36,6 +36,7 @@
     private static final Logger logger = LoggerFactory.getLogger(CodecSupport.class);
     private static Map<Byte, Serialization> ID_SERIALIZATION_MAP = new HashMap<Byte, Serialization>();
     private static Map<Byte, String> ID_SERIALIZATIONNAME_MAP = new HashMap<Byte, String>();
+    private static Map<String, Byte> Name_ID_MAP = new HashMap<String, Byte>();
```
Comment (beiwei30) [inline dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/CodecSupport.java]: **SERIALIZATIONNAME_ID_MAP** is better, in my opinion.

Comment (Jeff-Lv) [reply dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/CodecSupport.java]: good advice.

**Comment by beiwei30** [comment]
this pull request not valid any longer, has talked to the author.

**Pull request closed without merging**